web-registry/src/lib/queries/react-query/ecocredit/getAddDataToBatchesQuery/getAddDataToBatchesQuery.types.ts

```diff
@@ -0,0 +1,18 @@
+import { QueryClient, QueryObserverOptions } from '@tanstack/react-query';
+
+import { BatchInfoWithSupply } from 'types/ledger/ecocredit';
+import { AddDataToBatchesParams } from 'lib/ecocredit/api';
+
+import { ReactQueryBuilderResponse } from '../../types/react-query.types';
+
+export type ReactQueryAddDataToBatchesResponse = QueryObserverOptions<
```

Comment:
Using `QueryObserverOptions` here allows us to have all possible options for a query.